templates/bake/Command/command.twig

```diff
@@ -47,9 +47,10 @@ class {{ name }}Command extends Command
      *
      * @param \Cake\Console\Arguments $args The command arguments.
      * @param \Cake\Console\ConsoleIo $io The console io
-     * @return int|null|void The exit code or null for success
+     * @return int The exit code
      */
-    public function execute(Arguments $args, ConsoleIo $io)
+    public function execute(Arguments $args, ConsoleIo $io): int
```

Comment:
```suggestion
    public function execute(Arguments $args, ConsoleIo $io): ?int
```
Wouldn't this be more correct?

Reply:
I would rather we keep it `int` only so that in a future release we can add `int` type to our base class `Command::execute()` itself.

Reply:
Wouldnt this start with core though
Baked Code by no means would be a reliable way of going into that direction

Reply:
This kind of change can only be enforced in core in 6.x anyway.
With this change in bake it at least encourages new commands being generated to use this kind of setup.

Reply:
Is requiring an explicit return value for commands going to make userland code better? I can see how it makes the framework code simpler, but does it also benefit end users?

Reply:
I very much doubt that.
That said: Symfony went into that direction and the new versions require a non void (maybe even integer) return code.

My point here is that bake is the end of the chain - and narrowing it here to just int seems not like the best idea if there was no broader discussion on core changes yet.